### DISCUSSION TITLE
Help installing erlang_ls on debian

### DISCUSSION THREAD

**Discussion by b-sep** [mason-org/mason.nvim#1731] [Q&A]
Any one? Im using `debian12` and i have erlang and rebar3 installed, when i try to install `erlang_ls` throught mason ive got an error.

![Captura de tela de 2024-06-09 20-09-00](https://github.com/williamboman/mason.nvim/assets/65205717/28be6acb-530a-4a4c-958c-a5d46b332c05)
![Captura de tela de 2024-06-09 20-09-15](https://github.com/williamboman/mason.nvim/assets/65205717/754b9274-36a3-4544-8334-64aa33c14dae)
![Captura de tela de 2024-06-09 20-09-37](https://github.com/williamboman/mason.nvim/assets/65205717/8a2c78f7-cb1f-4eac-8f3f-71f83d51c0e2)

Looking on Mason status when i try to install, i can see that the error is on the `katana_code` compile, but outside mason (and neovim) i can pass that step just fine.

![image](https://github.com/williamboman/mason.nvim/assets/65205717/a381d0e4-3115-47db-9cce-6e6f0f263af3)
![image](https://github.com/williamboman/mason.nvim/assets/65205717/204349d6-64b7-416c-b793-9354977b8bb6)

The mason log just shows bash exit code status (127)...

**Comment by hologroove** [comment]
I am having the exact same problem on macOS Sonoma. I can build the `main` branch of `erlang_ls` with OTP 27 but not the `0.52.0` tag which Mason is trying to build. If I switch to OTP 26 then I can build `0.52.0`. 

I am managing OTP versions with `mise` so I created a `.mise.toml` in any directory with this content:
```
[tools]
erlang = "26"
```
Starting `nvim` from that directory will make `mason` to build `erlang_ls` with OTP 26 successfully.

**Reply by b-sep** [reply]
i ended up changing my otp version.
